### DISCUSSION TITLE
Sitemap Index

### DISCUSSION THREAD

**Discussion by rd23195** [patrickdemooij9/SeoToolkit.Umbraco#243] [Q&A] [answered]
Hi! I installed your sitemap package, I would just like to ask how can I get all the children of the page here when I go to the https:/homepage/sitemap.xml

![image](https://github.com/patrickdemooij9/SeoToolkit.Umbraco/assets/116734940/9d3d66cf-c091-493d-a8d3-222df7601539)

**Comment by patrickdemooij9** [comment]
Hi @rd23195 
That should happen automatically. Do you perhaps not have any children pages published in en-us?

**Reply by rd23195** [reply]
Hi @patrickdemooij9 thank you for the response let me rephrase the question, the user has multiple document types with Allow as root checked and they are used under the content tab, the question is can we get all of those document types/pages in one sitemap index? Btw, the homepage that I showed from my question at the top does not have any children.. is that why its just showing one urlset?

![image](https://github.com/patrickdemooij9/SeoToolkit.Umbraco/assets/116734940/8812cc47-5d03-4eb0-90a0-0d278c0e300e)

**Reply by patrickdemooij9** [reply]
Yes, how the sitemap generation currently works is that it will take the root item (homepage in this case) and then goes through all the children below it. That is why it only shows the homepage for you here.

I'll see what I can do for you here to make sure that the package supports your project layout as well.

**Reply by rd23195** [reply]
I see thank you for the explanation! Appreciate on checking to support the layout the we have we will wait for your response thanks again!

**Reply by patrickdemooij9** [reply, marked as answer]
Hi @rd23195, I just pushed an update that should be able to do this for you (https://github.com/patrickdemooij9/SeoToolkit.Umbraco/releases/tag/3.4.0).
The only thing you should do is change the Appsettings.json:
![image](https://github.com/patrickdemooij9/SeoToolkit.Umbraco/assets/11466511/930d6e23-efe4-437b-8309-19a9dd1826d0)

Let me know if this works for you!
